libs/ui/src/lib/test-status/test-status.component.html

```diff
@@ -0,0 +1,68 @@
+<div class="container">
+  <div class="inner-container">
+    <mat-list role="list">
+      <h3 matSubheader>
+        Status
+      </h3>
+      <div class="status-test_success">
+        <mat-list-item *ngIf="status?.buildProgress === 100">
+          <mat-icon matListIcon>check_circle</mat-icon>
+          <div class="status-content">
+            <div fxFill fxLayout="row" fxLayoutAlign="start center">
+              <div fxFlex>Build successful</div>
+            </div>
+          </div>
+        </mat-list-item>
+      </div>
+      <div [ngSwitch]="status?.testStatus" [ngClass]="statusClassName()">
+        <mat-list-item *ngSwitchCase="'test_pending'">
+          <mat-progress-spinner mode="indeterminate" [diameter]="18"></mat-progress-spinner>
+          Starting...
+        </mat-list-item>
+        <mat-list-item *ngSwitchCase="'test_inprogress'">
+          <mat-progress-spinner mode="indeterminate" [diameter]="18"></mat-progress-spinner>
+          <span *ngIf="status?.buildProgress === 100; then built; else building"></span>
+          <ng-template #building>
+            Building... ({{ status?.buildProgress }}%)
+          </ng-template>
+          <ng-template #built>
+            Running tests ({{ status?.success }}/{{ status?.total }})...
+          </ng-template>
+        </mat-list-item>
+        <mat-list-item *ngSwitchCase="'test_success'">
+          <mat-icon matListIcon>check_circle</mat-icon>
+          <div class="status-content">
+            <div fxFill fxLayout="row" fxLayoutAlign="start center">
+              <div fxFlex>All tests passed</div>
+            </div>
+          </div>
+        </mat-list-item>
+        <mat-list-item *ngSwitchCase="'test_failure'">
+          <mat-icon matListIcon>error</mat-icon>
+          <div class="status-content">
+            <div fxFill fxLayout="row" fxLayoutAlign="start center">
+              <div fxFlex>{{ status?.failure }} failed, {{ status?.success }} passed</div>
+            </div>
+          </div>
+        </mat-list-item>
+      </div>
+      <ng-container *ngIf="status?.errors.length > 0">
+        <mat-divider></mat-divider>
+        <h3 matSubheader>
+          Failed tests
+        </h3>
+        <mat-divider></mat-divider>
+        <mat-accordion>
+          <mat-expansion-panel *ngFor="let error of status?.errors">
```

Comment:
You might consider adding a track by here so that if some subset of errors remains the the corresponding dom elements stick around

Reply:
👍